backend/app/models.py

```diff
@@ -4,3 +4,32 @@
 from werkzeug.security import generate_password_hash, check_password_hash
 
 # create model classes here
+
+# Many-to-Many relationships requires helper table
+# According to the flask documentation
+contacts = db.Table(
+    "contacts",
+    db.Column("contact_id", db.Integer, db.ForeignKey("contact.id"), primary_key=True),
+    db.Column("project_id", db.Integer, db.ForeignKey("project.id"), primary_key=True),
+)
+# Project Model
+
+
+class Project(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    address = db.Column(db.String(256))
+    city = db.Column(db.String(64))
+    province = db.Column(db.String(64))
+    postalCode = db.Column(db.String(64))
+    neighbourhood = db.Column(db.String(256))
+    year = db.Column(db.Integer)
+    name = db.Column(db.String(64))
+    type = db.Column(db.String(64))
```

Comment:
I think a project should always be associated with a project type so this should have `nullable=false`

Reply:
note to self: ask Mu if they will have a set number of project types (in that case, we want an `Enum` column)

Reply:
update: we will most likely make project type editable so no `Enum` column